### PR TITLE
Added simple implementation of external events

### PR DESCRIPTION
**Pull request by KillerX**
as per https://projects.bcc.no/#/tasks/35687679

There is nothing actually "happening" to the data yet but it can be developed against if needed

### CODE REVIEW

File: backend/external-events/events.go [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+
+// ParseEvent from the json bytes
+func ParseEvent(data []byte) (*Event, error) {
+	e := &Event{}
+	err := json.Unmarshal(data, e)
+	if err != nil {
+		return nil, merry.Wrap(err)
+	}
+
+	if e.Type == TypeSong {
```
Comment (fredrikved) [inline backend/external-events/events.go:101]: Is a switch case more clean here?

File: backend/external-events/events.go [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+{
+	"timestamp": "2009-11-11T08:00:00+09:00", // RFC3339 formatted
+	"source": "some-system", // Arbitrary string identifying the source of the message, Informative
+	"type": "event.happened", // Arbitrary string, identifying the format of the data sent
+	"data": {"A": "B", "C": 99 }, // Arbitrary json object containing the actual data
+}
+*/
+
+// Event is the base event wihth data represented as raw json
+type Event struct {
```
Comment (fredrikved) [inline backend/external-events/events.go:21]: Just a thought.

Maybe "hide" BibleData, SongData, etc behind GetBibleData / GetSongData functions?
We have to check the type whenever we want to retrieve the event data, so might as well wait with the deserialization until we need it, or? That way, we don't need to make pointers, as we know they are not nil.

Comment (KillerX) [reply backend/external-events/events.go:21]: I'll actually wait with this as I don't know how this will be used internally yet.

Comment (KillerX) [reply backend/external-events/events.go:21]: Or, no I'll hide them, Makes it easier to not serialize to json.

**Pull request merged**